apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/test/__snapshots__/template-selector-control.test.js.snap

```diff
@@ -35,12 +35,6 @@ exports[`TemplateSelectorControl Basic rendering renders with required props 1`]
                 class="template-selector-item__media"
               />
             </div>
-            <span
```

Comment:
This span was removed here: https://github.com/Automattic/wp-calypso/pull/39120/files#